packages/sources/dlc-btc-por/README.md

```diff
@@ -0,0 +1,55 @@
+# DLC_BTC_POR
+
+![0.0.0](https://img.shields.io/github/package-json/v/smartcontractkit/external-adapters-js?filename=packages/sources/dlc-btc-por/package.json) ![v3](https://img.shields.io/badge/framework%20version-v3-blueviolet)
+
+This document was generated automatically. Please see [README Generator](../../scripts#readme-generator) for more info.
+
+## Environment Variables
+
+| Required? |         Name          |                                        Description                                        |  Type  |             Options             |                   Default                    |
+| :-------: | :-------------------: | :---------------------------------------------------------------------------------------: | :----: | :-----------------------------: | :------------------------------------------: |
+|    ✅     |        RPC_URL        |                          The RPC URL to connect to the EVM chain                          | string |                                 |                                              |
+|    ✅     |       CHAIN_ID        |                              The EVM chain id to connect to                               | number |                                 |                   `42161`                    |
+|    ✅     |     DLC_CONTRACT      |                        Contract address to fetch all funded vaults                        | string |                                 | `0x20157DBAbb84e3BBFE68C349d0d44E48AE7B5AD2` |
+|    ✅     |    BITCOIN_RPC_URL    |                                THE RPC URL of bitcoin node                                | string |                                 |                                              |
+|    ✅     |    BITCOIN_NETWORK    |                                   Bitcoin network name                                    |  enum  | `mainnet`, `regtest`, `testnet` |                  `mainnet`                   |
+|    ✅     |     CONFIRMATIONS     |                      The number of confirmations to query data from                       | number |                                 |                     `6`                      |
```

Comment:
Not sure if some of these should be env vars vs input params 🤔 
Particularly CONFIRMATIONS, I think it is typically a request param. I'd say lean towards whatever we do in other EAs

Reply:
You're right, vast majority of the EAs use `CONFIRMATIONS` input param, instead of config. There are however few reasons why i didn't want to have input params at all. 
1. In case of `CONFIRMATIONS` it makes sense to have it as request param if there are also addresses provided in the same request. For example having a generic indexer adapter that takes different addresses and returns a balance. In this case each address list that is given to indexer may require different confirmation number, so makes sense to also provide confirmations in the request. DLC.BTC  is not generic and fetches the addresses (vaults) internally. We still could have it in input params for consistency, but there are other concerns as well
2. In case of other properties like `DLC_CONTRACT`. Again, a lot of EAs have `contract` as a generic input param, but i think in this particular case it makes sene to have it in config. Generally i wanted to avoid input params because providing them with different values will just worsen the performance. For example in  a scenario when  Nops want to run one EA that supports both testnet and mainnet.  If we support `contract` or `confirmations` as input params, so that both chains can be used, then providing different values for those (two requests, one has `contract: MAINNET_CONTRACT`, the other has `contract: TESTNET_CONTRACT`) will actually slow down and possibly timeout the EA, as it will take much longer to process both params.  However, having those in config, now the Nops have to run another instance of EA for testnet. In this case each instance will deal maximum with 10000 requests, instead of 20000 and there is a less chance of EA timeout. Otherwise if we make `contract` as input param we would need to make `rpc_url` an input param as well.

We can still move `confirmations`, `contract` and probably others from config to input params, assuming some conditions are always true. Happy to read your suggestions or ideas.